mne/forward/tests/test_field_interpolation.py

```diff
@@ -158,6 +158,14 @@ def test_make_field_map_meg():
 
     assert_raises(ValueError, make_field_map, evoked, ch_type='foobar')
 
+    # now test the make_field_map on head surf for MEG
+    evoked.pick_types(meg=True, eeg=False)
+    fmd = make_field_map(evoked, trans_fname, meg_surf='head',
+                         subject='sample', subjects_dir=subjects_dir)
```

Comment:
This now requires `trans_fname`, which in turn requires the testing dataset, which in turn means this test needs the `requires_testing_data` decorator (it's why Travis is angry). Then we can merge!